openmc/data/neutron.py

```diff
@@ -327,19 +470,25 @@ def from_hdf5(cls, group_or_filename):
         mass_number = group.attrs['A']
         metastable = group.attrs['metastable']
         atomic_weight_ratio = group.attrs['atomic_weight_ratio']
-        temperature = group.attrs['temperature']
+        kTg = group['kTs']
+        kTs = []
+        for temp in kTg:
+            kTs.append(kTg[temp].value)
+        temperatures = [str(int(round(kT / K_BOLTZMANN))) + "K" for kT in kTs]
```

Comment:
I would actually make `temperatures` a property of `IncidentNeutron` that is computed automatically based on `self.kTs`. That way, you don't need to define it multiple times.